dkroutingtool/src/py/visualization.py

```diff
@@ -247,43 +245,13 @@ def createVisObject(coordArray):
                 “properties”: {{}}}},              
                 """.format(Node2Coords)
             )
-
-def create_geojson(assignment, routing, data):
-    """Creates geojson object from routing assignment
-    writes js file with geojson for Leaflet map (or any other js based map)
-    returns geojson dict object with feature collection of MultiLineString
-            each MultiLineString is a vehicle route
-    """
-    route_dict = create_route_dict(assignment, routing, data)
-
-    feature_collection_list = []
-
-    # Add MultiLineString geometries to feature collection
-    for vehicle_id in range(data.num_vehicles):
-        route_coordinates = [(route_dict[vehicle_id]["route"][i][0],
-                       route_dict[vehicle_id]["route"][i + 1][0]) for i in
-                       range(len(route_dict[vehicle_id]["route"]) - 1)]
-
-        route_geometry = MultiLineString(route_coordinates)  # type: MultiLineString
-        route_feature = Feature(geometry=route_geometry, id=vehicle_id)
-        feature_collection_list.append(route_feature)
-    feature_collection = FeatureCollection(feature_collection_list)
-
-    geojson = feature_collection # adding separate variable in case other features added
-
-    geojson_str = json.dumps(geojson) # creates json compact (indent can be added if needed)
-
-    output_filename = 'route_geojson.js'
-    with open(output_filename, 'w') as output_file:
-        output_file.write('var dataset = {};'.format(geojson_str))
-
-    return(geojson)
    
 def custom_sort(to_sort):
         to_sort = list(to_sort)
         if isinstance(to_sort[0], int):
             return sorted(to_sort)
         else:
+            return sorted(to_sort)
             return sorted(to_sort, key=lambda x: (int(x.split('-')[0]), x.split('-')[1]) if '-' in x else (int(x), 0))
```

Comment:
will clean up that obsolete line, the key argument is useless given the standard naming scheme